mod_lookup_identity.c

```diff
@@ -88,6 +99,19 @@ static int lookup_user_by_certificate(request_rec * r) {
 	dbus_error_init(&error);
 	DBusMessage * message = NULL;
 	DBusMessage * reply = NULL;
+#ifndef NO_CERTIFICATE_MAPPING_SUPPORT
+	const char * username;
+	apr_table_t * arg_table = NULL;
+
+	if (cfg->arg_name) {
+		ap_args_to_table(r, &arg_table);
+		if ((username = apr_table_get(arg_table, cfg->arg_name)) == NULL) {
+			username = "";
```

Comment:
Ah, you assign it here.
Alright, I think just `username = apr_table_get()` and then `if (! username) { username = "" }`.